lib/lens-frames.ts

```
@@ -0,0 +1,158 @@
+            return {
+                isValid: false,
+                error: 'Invalid Lens frame message',
+            };
+        }
+ 
+        const validatedPayload: FramePayloadValidated = {
+            object: 'validated_frame_action',
+            url: request.untrustedData.url,
+            interactor: {
+                object: 'user',
+                fid: parseInt(request.untrustedData.profileId), 
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)